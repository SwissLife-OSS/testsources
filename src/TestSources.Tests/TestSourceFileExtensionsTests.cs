using Xunit;
using FluentAssertions;
using TestSources.Interfaces;

namespace TestSources.Tests
{
    public class TestSourceFileExtensionsTests
    {
        [Fact]
        public void FileExtensionAsString_Succeeds()
        {
            // Arrange
            string fileName = "SomeJson.json";

            // Act
            ITestSourceFile file = TestSource.GetFile(fileName, true);
            string textInFile = file.AsString();

            string fileContentsAsString =
                TestSource.GetFile(fileName, true)
                .AsString();

            // Assert
            fileContentsAsString.Should().NotBeNullOrWhiteSpace();
        }

    }
}
