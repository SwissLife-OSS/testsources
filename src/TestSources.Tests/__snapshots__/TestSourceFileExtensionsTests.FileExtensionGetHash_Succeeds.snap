5DF3D751994310F3ACBE57252B8605634376D8B99C94310960511ADAC2DCE097
